tests/bsim/bluetooth/mesh/tests_scripts/dfu/dfu_cli_all_targets_lost_on_apply.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# The test instance sequence must stay as it is due to addressing scheme
```

Comment:
Minor. Please add test description to all files.

Reply:
done